src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_item.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="sf-toolbar-block sf-toolbar-block-{{ name }} sf-toolbar-status-{{ status|default('normal') }} {{ additional_classes|default('') }}">
-    {% if link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
+    {% if link is defined and link %}<a href="{{ path('_profiler', { token: token, panel: name }) }}">{% endif %}
```

Comment:
@xabbuh what do you think about rename `link` by `show_link`? `link` looks as though a URL is required to access of the profile panel IMO.